node/sc/bridge_accounts.go

```diff
@@ -70,8 +81,32 @@ func (ba *BridgeAccounts) GetBridgeOperators() map[string]interface{} {
 	return res
 }
 
+// GetParentOperatorFeePayer can return the fee payer of parent operator.
+func (ba *BridgeAccounts) GetParentOperatorFeePayer() common.Address {
+	return ba.pAccount.feePayer
+}
+
+// SetParentOperatorFeePayer can set the fee payer of parent operator.
+func (ba *BridgeAccounts) SetParentOperatorFeePayer(feePayer common.Address) error {
+	ba.pAccount.feePayer = feePayer
+	ba.db.WriteParentOperatorFeePayer(feePayer)
+	return nil
```

Comment:
It always returns `nil`, is it intended?

Reply:
Yes it is. There is no failed case, so this returns nil always.